modules/servers/openprovider/openprovider.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+if (!defined('WHMCS')) {
+    die('This file cannot be accessed directly');
+}
+
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'functions.php';
+
```

Comment:
please add some description for this file
how it's used? add some links from whmcs documentation
because it's not clear why this file is just set of functions and how this functions are called